FWCore/Framework/bin/cmsRun.cpp

```diff
@@ -240,6 +240,8 @@ int main(int argc, char* argv[]) {
       try {
         std::unique_ptr<edm::ParameterSet> parameterSet = edm::readConfig(fileName, argc, argv);
         processDesc.reset(new edm::ProcessDesc(std::move(parameterSet)));
+      } catch (edm::Exception const&) {
+        throw;
```

Comment:
I wonder if adding the calling context to the exception would be helpful to interpret the exception message?

Reply:
The call actually does have a lot of context. E.g.
```
----- Begin Fatal Exception 14-Dec-2022 14:21:44 CST-----------------------
An exception of category 'Configuration' occurred while
   [0] Processing the python configuration file named test_pyhon_error.py
Exception Message:
test message

At:
  test_pyhon_error.py(11): <module>
----- End Fatal Exception -------------------------------------------------
```

for my trivial test case where the configuration file only has the `raise` call.